docs/data/index.md

```diff
@@ -0,0 +1,211 @@
+
+<details>
+  <summary>Line Items export example</summary>
+  <div>
+    <pre><code>
+Order ID,Order Number,External Order ID,Cart ID,Product ID,Variant ID,Quantity Sold,Product Name,Merchant Name,Order Timestamp,Goods Without Tax<br></br>
+clo925si20000azzodbeggfv7,1001,EXT1001,3aasunj099y9,cks0fz5eo00001sr269h63310,0400151858640,2,"Widget A, Size: L",MerchantX,2023-10-25T14:31:21.000Z,$40.00<br></br>
+clo925si20000azzodbeggfv7,1001,EXT1001,3aasunj099y9,clbdx2e2l0383011h2kzx8cjp,40427980980431,1,Widget B,MerchantX,2023-10-25T14:31:21.000Z,$50.00
+    </code></pre>
+  </div>
+</details>
+
+
+#### Line Item Fields
+
+| Field            | Description                                                         |
+| :--------------- | :------------------------------------------------------------------ |
+| Order ID         | Unique identifier for the order                                     |
+| Order Number     | Human-readable order number                                         |
+| External Order ID| External reference ID for the order, if provided, otherwise "empty" |
+| Cart ID          | Unique identifier for the cart, or "empty"                          |
+| Product ID       | Unique identifier for the product                                   |
+| Variant ID       | Unique identifier for the product variant                           |
+| Quantity Sold    | Number of units sold                                                |
+| Product Name     | Name of the product                                                 |
+| Merchant Name    | Name of the merchant                                                |
+| Order Timestamp  | Time when the order was placed in UTC, in ISO 8601 format           |
+| Goods Without Tax| Cost of goods sold without tax                                      |
+
+### Orders Export
+
+The Orders export provides an overview of all orders processed.
+
+<details>
+  <summary>Orders export example</summary>
+  <div>
+    <pre><code>
+Order ID,Order Number,External Order ID,Cart ID,Order Timestamp,Total Customer Charge,Goods Without Tax,Goods Tax,Shipping Without Tax,Shipping Tax,Duties<br></br>
+clo925si20000azzodbeggfv7,1001,EXT1001,3aasunj099y9,2023-10-25T14:31:21.000Z,$100.00,$90.00,$10.00,$0.00,$0.00,$0.00
+    </code></pre>
+  </div>
+</details>
+
+#### Order Fields
+
+| Field                 | Description                                                                   |
+| :-------------------- | :---------------------------------------------------------------------------- |
+| Order ID              | The unique identifier for the order, generated by Bonsai                      |
+| Order Number          | A human-readable order number                                                 |
+| External Order ID     | Identifier from an external system, if provided, otherwise "empty"            |
+| Cart ID               | Unique identifier for the cart related to this order, or "empty"              |
```

Comment:
This is a major nitpick so feel free to disregard it, but I think consistency with regards to having an article or not would be a good improvement. So my suggestion is either remove `The` and `A` from the first two lines or add them to the bottom 2 lines. It seems like most descriptions don't have articles so maybe removing would be easier